tests/ci-tests/google-marketplace/.cirrus.expected.yml

```diff
@@ -0,0 +1,338 @@
+aws_credentials:
+  region: eu-central-1
+  role_arn: arn:aws:iam::275878209202:role/CirrusCI-10
+  role_session_name: cirrus
+ec2_instance:
+  subnet_filters:
+  - name: tag:Product
+    values:
+    - CirrusCI
+  - name: tag:EnvType
+    values:
+    - prod
+  - name: tag:EnvName
+    values:
+    - prod
+  - name: tag:version
+    values:
+    - 5
+  - name: tag:aws-cdk:subnet-name
+    values:
+    - PrivateSubnet
+eks_container:
+  builder_subnet_filters:
+  - name: tag:Product
+    values:
+    - CirrusCI
+  - name: tag:EnvType
+    values:
+    - prod
+  - name: tag:EnvName
+    values:
+    - prod
+  - name: tag:version
+    values:
+    - 5
+  - name: tag:aws-cdk:subnet-name
+    values:
+    - PrivateSubnet
+  nodeSelectorTerms:
+  - matchExpressions:
+    - key: node.kubernetes.io/instance-type
+      operator: In
+      values:
+      - m7a.8xlarge
+      - c7a.8xlarge
+      - r7a.8xlarge
+env:
+  CI_BUILD_NUMBER: "1"
+  CIRRUS_AGENT_VERSION: 1.136.0
```

Comment:
this implies that we will have to update our CI each time `CIRRUS_AGENT_VERSION` change ? same as any other devInfra changes ? this looks like a dependency lock.

could you explain the reasoning on addind the expected.yml in our CI ? i have a hard time figuring out benefits in our case.

Reply:
This file was auto-generated from `cirrus internal test`. I don't know where these values come from.

Reply:
in that case i am not sure if we want those tests and if they make sense for us, we should own everything.